src/software/pipeline/main_prepareDenseScene.cpp

```diff
@@ -90,7 +90,9 @@ bool prepareDenseScene(const SfMData& sfmData,
     //we have a valid view with a corresponding camera & pose
     const std::string baseFilename = std::to_string(viewId);
 
-    oiio::ParamValueList metadata = image::getMetadataFromMap(view->getMetadata());
+    // get metadata from source image to be sure we get all metadata (type conversion problems with string maps)
```

Comment:
`// get metadata from source image to be sure we get all metadata. We don't use the metadatas from the Views inside the SfMData to avoid type conversion problems with string maps.`